diff --git a/pygccxml/declarations/calldef.py b/pygccxml/declarations/calldef.py
index fd1fe17..7ed66bd 100644
--- a/pygccxml/declarations/calldef.py
+++ b/pygccxml/declarations/calldef.py
@@ -90,6 +90,11 @@ class argument_t(object):
     def __lt__(self, other):
         if not isinstance(other, self.__class__):
             return self.__class__.__name__ < other.__class__.__name__
+        if other.default_value == None:
+            return False
+        if self.default_value == None:
+            return self.name < other.name \
+                and self.decl_type < other.decl_type
         return self.name < other.name \
             and self.default_value < other.default_value \
             and self.decl_type < other.decl_type
